fenix/app/src/main/java/org/mozilla/fenix/compose/list/ListItem.kt

```diff
@@ -200,6 +206,54 @@ fun IconListItem(
     )
 }
 
+/**
+ * List item used to display a label with an optional description text and
+ * an [RadioButton] at the beginning.
+ *
+ * @param label The label in the list item.
+ * @param modifier [Modifier] to be applied to the layout.
+ * @param description An optional description text below the label.
+ * @param maxDescriptionLines An optional maximum number of lines for the description text to span.
+ * @param onClick Called when the user clicks on the item.
+ * @param selected [Boolean] That indicates whether the [RadioButton] is currently selected.
+ */
+@Composable
+fun RadioButtonListItem(
+    label: String,
+    modifier: Modifier = Modifier,
+    description: String? = null,
+    maxDescriptionLines: Int = 1,
+    onClick: (() -> Unit),
+    selected: Boolean = false,
```

Comment:
it'll probably make more sense to have this as a mandatory parameter of the API without the optional boolean. That way we can always ensure all downstream consumers always give it a value.

Reply:
Done